framework/src/modules/chain/chain.js

```diff
@@ -319,6 +307,53 @@ module.exports = class Chain {
 		this.logger.info('Cleaned up successfully');
 	}
 
+	async _initModules() {
```

Comment:
Can't we move all this logic to `./init_steps/init_modules`?

Reply:
we can move to the init_modules file, but it will not be dynamic anymore.
so probably it's easier to access all the required parameters here